R/fb_team_match_stats.R

```diff
@@ -0,0 +1,166 @@
+                                     Home_Tackles = as.numeric(.data[["Home_Tackles"]]),
+                                     Home_Interceptions = as.numeric(.data[["Home_Interceptions"]]),
+                                     Home_Aerials_Won = as.numeric(.data[["Home_Aerials_Won"]]),
+                                     Home_Clearances = as.numeric(.data[["Home_Clearances"]]),
+                                     Home_Offsides = as.numeric(.data[["Home_Offsides"]]),
+                                     Home_Goal_Kicks = as.numeric(.data[["Home_Goal_Kicks"]]),
+                                     Home_Throw_Ins = as.numeric(.data[["Home_Throw_Ins"]]),
+                                     Home_Long_Balls = as.numeric(.data[["Home_Long_Balls"]]),
+                                     Away_Possession = as.numeric(.data[["Away_Possession"]]),
+                                     Away_Success_Pass = as.numeric(.data[["Away_Success_Pass"]]),
+                                     Away_Pass = as.numeric(.data[["Away_Pass"]]),
+                                     Away_Passing_Accuracy = as.numeric(.data[["Away_Passing_Accuracy"]]),
+                                     Away_Fouls = as.numeric(.data[["Away_Fouls"]]),
+                                     Away_Corners = as.numeric(.data[["Away_Corners"]]),
+                                     Away_Crosses = as.numeric(.data[["Away_Crosses"]]),
+                                     Away_Touches = as.numeric(.data[["Away_Touches"]]),
+                                     Away_Tackles = as.numeric(.data[["Away_Tackles"]]),
+                                     Away_Interceptions = as.numeric(.data[["Away_Interceptions"]]),
+                                     Away_Aerials_Won = as.numeric(.data[["Away_Aerials_Won"]]),
+                                     Away_Clearances = as.numeric(.data[["Away_Clearances"]]),
+                                     Away_Offsides = as.numeric(.data[["Away_Offsides"]]),
+                                     Away_Goal_Kicks = as.numeric(.data[["Away_Goal_Kicks"]]),
+                                     Away_Throw_Ins = as.numeric(.data[["Away_Throw_Ins"]]),
+                                     Away_Long_Balls = as.numeric(.data[["Away_Long_Balls"]]))
+    )
+
+
+  } else {
+    print(glue::glue("{Game_URL} is not available"))
+    each_game <- data.frame()
+  }
+
+  return(each_game)
+}
+
+#' Get FBref match team stats
+#'
+#' Returns match team stats for selected matches.
+#'
+#' @param match_url the fbref.com URL for the required match
+#' @param time_pause the wait time (in seconds) between page loads
+#'
+#' @return returns a dataframe with the match team stats for a selected match
+#'
+#' @importFrom magrittr %>%
+#' @importFrom rlang .data
+#'
+#' @export
+#'
+#' @examples
+#' \dontrun{
+#' try({
+#' match <- fb_match_urls(country = "AUS", gender = "F", season_end_year = 2021, tier = "1st")[1]
+#' df <- fb_team_match_stats(match_url = match)
```

Comment:
this should be updated to match the function

Reply:
Can you please action this @shufinskiy?

Reply:
@JaseZiv

I'm not a native English speaker, so I didn't quite understand, I'll try to clarify:

Do both functions remain? From the comment above, I did understand that you do not think such a function (understat_available_teams) is necessary.

If both remain, then I will make the edits that I highlighted @tonyelhabr

Reply:
@JaseZiv @tonyelhabr

Reply:
ah ok, on second review, this looks fine. my eyes only saw the call to `fb_match_urls()`, but i see that you then feed the returned match url into your new function `fb_team_match_stats()`. (i overlooked the call to `fb_team_match_stats()` before.)